oak/server/rust/oak_abi/build.rs

```diff
@@ -15,11 +15,17 @@
 //
 
 fn main() {
-    oak_utils::run_protoc_rust(protoc_rust::Args {
-        out_dir: "src/proto",
-        input: &["../../../../oak/proto/oak_api.proto"],
-        includes: &["../../../../oak/proto"],
-        customize: protoc_rust::Customize::default(),
-    })
-    .expect("protoc");
+    // Tell Cargo that if the given file changes, to rerun this build script.
+    println!("cargo:rerun-if-changed=../../../../oak/proto/oak_api.proto");
```

Comment:
I'm not sure it makes a difference, but I would move this line after the actual code generation, so if things go wrong it will not have printed something incorrect.

Reply:
Ah cool, thanks for the heads-up, fine to keep at the top then, let's make sure this is consistent across the various build scripts anyways (cc @conradgrobler ).